test/verify/run_verify.cpp

```diff
@@ -168,7 +168,7 @@ void run_verify::verify(const std::string& name, const migraphx::program& p) con
     std::vector<std::string> target_names;
     for(const auto& tname : migraphx::get_targets())
     {
-        if(tname == "ref")
+        if(tname == "ref" || tname == "fpga")
```

Comment:
Is the plan to remove this exception once we integrate fpga implementations for now? I think this will remove the CI checks for test_compile

Reply:
Yes, that is the eventual plan once it's actually functional. The test I added does pass but the verify tests try to run different ops on all targets and those don't work

Reply:
Similar to something we do for disabled_tests, it looks like you can stub out the other targets based on target name. Do this for your added test case to add it to CI, and that should be sufficient. We can slowly add further tests for the other targets as you build this out.

My concern with stubbing things out for all tests, is that we're not all testing the latest and greatest then should any issues arise with new changes from develop and your fpga changes as you integrate them in.

Add a todo comment as part of your disabled_tests added for stuff covered in "fpga" and then we can open an issue after this PR for the other targets/pieces to keep track of things.

Reply:
Are you referring to [this ](https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/blob/develop/test/verify/main.cpp#L70=)for disabled_tests? If so, I understood that as a blacklist to prevent specific tests from running. What I need is to prevent all these verify tests from running because none of these operations will work right now. The test case I added is already part of the CI since it's outside the verify/. At least, I can see it running when I do `make check` locally. Perhaps I'm misunderstanding your request.

I can add a TODO for this here, sure.

Reply:
Got it, thanks for the reference. I think adding this to the CI may be a little premature at this point since we'll be working in the coming months with other teams in China to help create it. Once it's in a little more stable state, we can test it regularly. At this point, it's intended more as placeholder code so the teams in China have something to look at and fill in since you can't actually call `eval` on something targeting the FPGA. Do you feel otherwise?

I've updated the TODO.

Reply:
My initial plan was to just let developers run any FPGA tests locally since they will not be making frequent PRs to this repo anyway but okay, I've added it to the CI.